app/policies/hbx_profile_policy.rb

```diff
@@ -337,6 +337,11 @@ def can_submit_time_travel_request?
     role.permission.can_submit_time_travel_request
   end
 
+  def super_admin?
+    role = user_hbx_staff_role
+    role&.permission&.name == "super_admin"
+  end
```

Comment:
@vkghub As discussed, let's refactor this to not depend on the `name` of the permission and do one of the below:
- Find an existing field that is only set to true for Super Admins that we could use **(OR)**
- Add a new field to the permission model to support the `IVL Dry Run Dashboard` endpoint.